apps/frontend/src/components/generic/ColumnHeader.vue

```diff
@@ -32,36 +31,34 @@ export default class ColumnHeader extends Vue {
    * Toggles between the sort modes, and emits them as an "input" event.
    */
   toggle_sort(): void {
-    let new_sort: string;
+    let newSort: string;
     switch (this.sort) {
-      default: // Shouldn't happen but whatever
-      case 'none':
-        new_sort = 'descending';
-        break;
       case 'descending':
-        new_sort = 'ascending';
+        newSort = 'ascending';
         break;
       case 'ascending':
-        new_sort = 'descending';
+        newSort = 'descending';
+        break;
+      default: // Shouldn't happen but whatever
+        newSort = 'descending';
```

Comment:
Is there a need for `case 'ascending'` and `default`? Can't you just use `default`?